### ISSUE TITLE
Spell checker module not loaded

### ISSUE THREAD

**Issue by lasombra** [flathub/chat.rocket.RocketChat#13]
I'm using Fedora 29 workstation. All packages up-to-date.

When running Rocket.Chat using `flatpak run --env=ELECTRON_ENABLE_LOGGING=1 chat.rocket.RocketChat` I see the following stack trace:
```
[18:0312/071415.869412:INFO:CONSOLE(159)] "Error: /app/Rocket.Chat/node_modules/spellchecker/build/Release/spellchecker.node: undefined symbol: _ZN2v816FunctionTemplate3NewEPNS_7IsolateEPFvRKNS_20FunctionCallbackInfoINS_5ValueEEEENS_5LocalIS4_EENSA_INS_9SignatureEEEiNS_19ConstructorBehaviorE
    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:160:31)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:722:18)
    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:160:31)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/app/Rocket.Chat/node_modules/spellchecker/lib/spellchecker.js:3:16)
    at Object.<anonymous> (/app/Rocket.Chat/node_modules/spellchecker/lib/spellchecker.js:109:3)", source: /app/Rocket.Chat/node_modules/electron/dist/resources/electron.asar/renderer/init.js (159)
```

This doesn't happen when running the RPM version.

**Comment by andrevmatos** [comment]
tl;dr: back to pre-built packages, which solve this
These symbols errors started happening I think since 2.14.1 or 2.15.0, when RocketChat upgraded to Electron 4.x. Even being completely sure I was compiling the same node version on the Flatpak, `10.11.0`, these symbol errors wouldn't go away.
I just pushed update to 2.15.1, which go back to pre-built binaries, so we get the released and stable binaries from RocketChat. When it becomes easier to package node/electron apps with Flatpak without having to list a ton of apps which breaks every now and then, we can go back to source build.

**Comment by lasombra** [comment]
That's working now, thanks!
